packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/loadingaconfiguration.md

```diff
@@ -0,0 +1,50 @@
+
+Create a configuration by calling `IdxClientConfiguration`. You can set the values
+directly in your code or load them from a file. This code shows loading the values from a
+property file in your project.
+
+First, create a property file, for example, `okta.properties` in the project root. Add the values for your Okta Application Integration to the file.
```

Comment:
For tutorial simplicity sake, we should not show them how to load this via gradle, we should encourage them to hardcode the values!

Reply:
I agree that hardcoded has less code, but it also isn't as generic, i.e., easy to cut/paste and modify. The typical solutions for such sample code is using a set of constants (or a const structure or similar), or using a property file. The latter has the advantage that it's easily modifiable. In iOS, it also dovetails nicely with the code to save a modified version of the configuration values (useful during development.)

Based on that, are you OK with a property list implementation?

Reply:
@danielmaharry-okta, I've added a reference to that page in the table description of the scopes value (index.md line 78)

Reply:
@deepu105, I agree and am keeping both the Android and iOS versions.